### PR TITLE
Attributes extractor skeleton

### PR DESCRIPTION
**Pull request by lgozman**
- define structure of the project
- add metrics
- add swagger configuration for doc generation
- add deployment files

### CODE REVIEW

File: src/main/java/ch/ricardo/attributeextractor/configuration/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RestController;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+/**
```
Comment (lgozman) [inline src/main/java/ch/ricardo/attributeextractor/configuration/SwaggerConfig.java]: haha, I will remove this

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	</parent>
+	<groupId>ch.ricardo.attributeextractor</groupId>
+	<artifactId>attribute-extractor-api</artifactId>
+	<version>0.0.1</version>
+	<name>attribute-extractor-api</name>
+	<description>Attribute extractor API</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<java-metrics-utils.version>1.1.0</java-metrics-utils.version>
```
Comment (ivannikolic) [inline pom.xml]: Remove this dependency. 
`java-metrics-utils` evolved into `ricardo-spring-metrics-starter` and I see you are using that one as well.

File: src/main/resources/logback-spring.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <logger name="ch.ricardo" level="${APP_LOGGING_LEVEL:-INFO}" additivity="false">
+    <appender-ref ref="STDOUT"/>
+  </logger>
+
+  <logger name="org.springframework.boot.context" level="INFO"/>
+
+  <root level="WARN">
+    <appender-ref ref="STDOUT"/>
+  </root>
+</configuration>
```
Comment (ivannikolic) [inline src/main/resources/logback-spring.xml]: It should become a standard adding a simple configuration for local runs, I think we should add it here as well.

Example: https://github.com/ricardo-ch/search-query-api-articles/blob/master/src/main/resources/logback-spring.xml#L68

File: pom.xml (hunk 2 above)
Comment (lgozman) [reply pom.xml]: ok

File: src/main/resources/logback-spring.xml (hunk 3 above)
Comment (lgozman) [reply src/main/resources/logback-spring.xml]: ok

**Pull request merged**